terraform.json.example

```diff
@@ -1,6 +1,8 @@
 {
   "cloudwatch_retention": 7,
-  "cluster_autoscaler_helm_config": {},
+  "cluster_autoscaler_helm_config": {
+    "version": "9.28.0"
```

Comment:
Why the version = "9.28.0" is changed with "version": "9.28.0"? What does this do for deployment?